### PR TITLE
Improve makefile and README.md 

### PR DESCRIPTION
**Pull request by cyrilleverrier**

### CODE REVIEW

**Comment by ruflin** [comment]
Looks like we work on similar things in parallel. I plan to merge yours first as I think it will solve some of my issues and rebase on top of it: https://github.com/elastic/beat-generator/pull/2 Let me know when it is ready for review.

File: {{cookiecutter.beat}}/Makefile [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
 
-# Path to the libbeat Makefile
-include vendor/github.com/elastic/beats/libbeat/scripts/Makefile
+PREFIX?=.
+ES_BEATS=./vendor/github.com/elastic/beats
+GOPACKAGES=$(shell glide novendor)
+include $(ES_BEATS)/libbeat/scripts/Makefile
+
+.PHONY: generate
+generate:
```
Comment (ruflin) [inline {{cookiecutter.beat}}/Makefile]: I hope we can integrate that into the standard makefile in the future as we plan to unify the template scripts below.

File: {{cookiecutter.beat}}/README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # {{cookiecutter.beat|capitalize}}
 
-Welcome to {{cookiecutter.beat}}.
+ Welcome to {{cookiecutter.beat|capitalize}}.
```
Comment (ruflin) [inline {{cookiecutter.beat}}/README.md]: Is the space in front needed?

Comment (cyrilleverrier) [inline {{cookiecutter.beat}}/README.md]: No, it's not needed. A typo error.

Kind regards,

Cyrille Verrier - Senior Software Developer
Mobile: +47 46227001
Fabriscale Technologies AS, http://fabriscale.com

File: {{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2 [hunk 3]
```diff
@@ -70,10 +70,10 @@ shipper:
 
   #files:
     # The directory where the log files will written to.
-    #path: /var/log/mybeat
+    #path: /var/log/{{cookiecutter.beat}}
```
Comment (ruflin) [inline {{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2]: To make this work, the ignoring of j2 files has to be removed from the cookiecutter.json file.

File: {{cookiecutter.beat}}/README.md (hunk 2 above)
Comment (cyrilleverrier) [inline {{cookiecutter.beat}}/README.md]: No, it's not needed. A typo error :)

File: {{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2 (hunk 3 above)
Comment (cyrilleverrier) [inline {{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2]: oh, you are right here. I haven't checked the output of that file.

**Comment by ruflin** [comment]
@cyrilleverrier It would be nice if you could apply the changes above and squash the commits. As soon as this is merged I will progress on https://github.com/elastic/beat-generator/pull/2

**Comment by ruflin** [comment]
@cyrilleverrier Thanks, merged. Will now continue on https://github.com/elastic/beat-generator/pull/2

**Pull request merged**